.cursor/Dockerfile

```
@@ -0,0 +1,22 @@
+# git and curl are required for repository checkout and tooling; ca-certificates
+# lets npm reach the registry over HTTPS.
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        curl \
+        git \
+    && rm -rf /var/lib/apt/lists/*
+
+# Bun is installed via its official npm package so the binary lands in the
+# global prefix (/usr/local/bin) and is on PATH for the node user.
+RUN npm install -g bun \
```

Hadolint (2.15.1)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)